### PR TITLE
Update Sol-Tec Users to Content and Cloud and renew access

### PR DESCRIPTION
**Pull request by HughSolTec**

### CODE REVIEW

**Review by ben-al — APPROVED** [review]
LGTM

File: terraform/MOJ-PTTP-DevicesAndApps-Pipeline-Windows10Apps.tf [hunk 1]
```diff
@@ -1,16 +1,55 @@
 module "MOJ-PTTP-DevicesAndApps-Pipeline-Windows10Apps" {
   source     = "./modules/repository-collaborators"
   repository = "MOJ.PTTP.DevicesAndApps.Pipeline.Windows10Apps"
-  collaborators = [
```
Comment (ben-al) [inline terraform/MOJ-PTTP-DevicesAndApps-Pipeline-Windows10Apps.tf:4]: This has to be present

**Review by ben-al — CHANGES_REQUESTED** [review]

File: terraform/MOJ-PTTP-DevicesAndApps-Pipeline-Windows10Apps.tf (hunk 1 above)
Comment (HughSolTec) [reply terraform/MOJ-PTTP-DevicesAndApps-Pipeline-Windows10Apps.tf]: Have updated . thank you :)

**Comment by ben-al** [comment]
@HughSolTec Morning mate - looks like some S3 credentials have expired or something, looking into this now. Apologies for the blocker.

**Comment by ben-al** [comment]
@HughSolTec Hi Hugh - i've just been informed that PRs from forks do not work, could you PR from branch please? Incredibly sorry for the inconvenience, I will update the docs to make this more clear.

**Comment by HughSolTec** [comment]
> 

Hi Ben, I don't think I have permissions to create a new branch as this is what I initially tried before forking. many thanks

**Comment by ben-al** [comment]
Ah ok - i'll make it for you, will tag you in it for confirmation.

**Comment by ben-al** [comment]
I merged a branch PR for you: https://github.com/ministryofjustice/github-collaborators/pull/380

Let me know if there are any problems with it, will close this.

**Pull request closed without merging**